src/components/container/SurveyTaskList/SurveyTaskList.stories.tsx

```diff
@@ -33,14 +95,29 @@ export const Sequential = {
 		...Incomplete.args,
 		sequential: true
 	},
+	argTypes: { ...Incomplete.argTypes },
```

Comment:
It looks like the use of `...Incomplete.args` here was pre-existing, but rather than borrowing from a prior scenario, it probably makes more sense to pull these reusable configuration pieces out into their own const variables.  Otherwise, if someone intends to change only the `Incomplete` story setup, they may not realize they are breaking the other ones.